app/controllers/concerns/exception_handler.rb

```diff
@@ -7,24 +7,12 @@ module ExceptionHandler
       json_response({ :message => e.message }, :not_found)
     end
 
-    rescue_from ActiveRecord::RecordInvalid do |e|
-      json_response({ :message => e.message }, :unprocessable_entity)
-    end
-
-    rescue_from ActionController::ParameterMissing do |e|
-      json_response({ :message => e.message }, :unprocessable_entity)
-    end
-
-    rescue_from ManageIQ::API::Common::EntitlementError, Exceptions::NotAuthorizedError do |e|
+    rescue_from ManageIQ::API::Common::EntitlementError do |e|
```

Comment:
@hsong-rh Can't we get rid of all the rescue_froms and have the common gem handle it. And we can add the exceptions to custom_exception_mappings. The common gem can handle nested exceptions, since each exception has a cause attribute which allows us to link exceptions together.

Reply:
@mkanoor I changed to use manageiq-api-common 2.1 to handle all exceptions.